src/commands/ghCommands/status.js

```diff
@@ -0,0 +1,68 @@
+const GitHubApi = require('github');
+const chalk = require('chalk');
+const github = GitHubApi();
+
+module.exports = {
+	command: 'status',
+	description: 'Set a PR status',
+	builder: yarg =>
+		yarg
+			.options({
+				value: {
+					alias: 's', // s for 'status'. -v is conventionally a 'version' alias
+					demandOption: true,
+					type: 'string',
+					describe: 'The status value',
+					choices: ['pending', 'failure', 'error', 'success'],
+				},
+				commit: {
```

Comment:
Since this needs to be a full SHA-1, might as well add a description with that info here.

Reply:
Good call - added descriptions to the other fields as well